File: demo/src/main/java/com/example/android/entity/PurchaseRequest.java
package com.example.android.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("purchase_requests")
public class PurchaseRequest {
    @TableId(type = IdType.AUTO)
    private Integer id;
    private Long buyerId;
    private String title;
    private String description;
    private Integer categoryId;
    private Double maxPrice;
    private Integer status;
    private Integer viewCount;
    private Date createdAt;
    private Date updatedAt;
}

File: app/src/main/java/com/example/xiaoyi/model/PurchaseRequest.kt
package com.example.xiaoyi.model

data class PurchaseRequest(
    val id: Long,
    val title: String,//求购标题
    val maxPrice: Double,//最高价格
    val description: String,//求购描述
    val categoryId: Int,//分类ID
    val buyerId: Long,//买家ID
    val status: Int,//状态
    val viewCount: Int,//浏览次数
    val createdAt: String//创建时间
)